.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "rules": {
+        "indent": ["error", 2],
+        "linebreak-style": ["error", "windows"],
+        "semi": ["error", "always"],
+		"brace-style": ["error", "stroustrup"],
+		"curly": ["error"]
```

Comment:
There seems still a problem with tabs and spaces

Reply:
Sorry for that, I hope I caught them all now.

Reply:
Nope, still wrong in this file